.checkstyle

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<fileset-config file-format-version="1.2.0" simple-config="false" sync-formatter="false">
+  <local-check-config name="maven-checkstyle-plugin checkstyle" location="file:/Users/lucas/git/Open-Realms-of-Stars/buildtools/checkstyle/ruleset.xml" type="remote" description="maven-checkstyle-plugin configuration checkstyle">
```

Comment:
This ".checkstyle" is probably for Eclipse's checkstyle plugin configuration file, right? It contains your local username paths. I assume this is something you do not want to add.